OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -266,12 +276,12 @@ public Actor GetActorBelow()
 
 		protected void ReserveSpawnBuilding()
 		{
-			/* HACK: not spawning in the air, so try to assoc. with our afld. */
-			var afld = GetActorBelow();
-			if (afld == null)
+			// HACK: Not spawning in the air, so try to associate with our spawner.
```

Comment:
Idea for a future PR: check for `ParentActorInit` in the constructor and drop this hack.